zitilib/models/characterization/distribution.go

```diff
@@ -28,11 +28,10 @@ func newDistributionFactory() model.Factory {
 }
 
 func (f *distributionFactory) Build(m *model.Model) error {
-	m.Distribution = model.DistributionBinders{
-		func(m *model.Model) model.DistributionStage { return distribution.Locations(models.ControllerTag, "logs") },
-		func(m *model.Model) model.DistributionStage { return distribution.Locations(models.RouterTag, "logs") },
-
-		func(m *model.Model) model.DistributionStage { return rsync.Rsync() },
+	m.Distribution = model.DistributionStages{
+		distribution.Locations(models.ControllerTag, "logs"),
+		distribution.Locations(models.RouterTag, "logs"),
+		rsync.Rsync(),
```

Comment:
We probably don't need to put most of these declarations into separate factories anymore... because they're mostly just static lists of stage implementations, we can probably either just enumerate them directly in the static model, or just create a single factory to declare all of them in one place.

Reply:
I was thinking that too, I'll try a couple of things and see what feels right. I'll do that in a follow up PR